lib/howl/ui/theme.moon

```diff
@@ -226,7 +226,7 @@ with config
   .define
     name: 'font'
     description: 'The main font used within the application'
-    default: 'Liberation Mono'
+    default: if ffi.os == 'OSX' then 'Monaco' else 'Liberation Mono'
```

Comment:
This could be a comma separated list instead, e.g. `Liberation Mono,Monaco` without any OS checking. Also, should be `Monaco` or `Menlo` be preferred? @bunster?